openmc/mgxs_library.py

```diff
@@ -1634,6 +1681,286 @@ def set_inverse_velocity_mgxs(self, inverse_velocity, temperature=294.,
         self._inverse_velocity[i] = inverse_velocity.get_xs(
             nuclides=nuclide, xs_type=xs_type, subdomains=subdomain)
 
+    def convert_representation(self, target_representation, num_polar=None,
+                               num_azimuthal=None):
+        """Produce a new XSdata object with the same data, but converted to the
+        new representation (isotropic or angle-dependent).
+
+        This method cannot be used to change the number of polar or
+        azimuthal bins of an XSdata object that already uses an angular
+        representation. Finally, this method simply uses an arithmetic mean to
+        convert from an angular to isotropic representation; no flux-weighting
+        is applied and therefore the correctness of the solution is not
```

Comment:
Considering making this statement of caution a tad bit stronger: "correctness of the solution is not guaranteed" -> "reactions rates will not be preserved".